docs/_includes/css/index-ie.css

```diff
@@ -0,0 +1,53 @@
+main.Homepage-Hero {
```

Comment:
`@media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {` this is what we need to do instead of conditionals.